cmd/flarectl/flarectl.go

```diff
@@ -378,7 +427,7 @@ func main() {
 	app := cli.NewApp()
 	app.Name = "flarectl"
 	app.Usage = "CloudFlare CLI"
-	app.Version = "2015.12.0"
+	app.Version = "2016.4.0"
```

Comment:
Right, I was thinking more in the scope of the library itself, I guess we don't really have any versioning for it right now?